docs/bcfusion/validators/slash.md

```diff
@@ -0,0 +1,52 @@
+# Slash
+
+The BNB smart chain (BSC) is a blockchain network that aims to provide fast, secure, and reliable transactions. To
+achieve this, the BSC relies on a set of validators who are responsible for producing and validating blocks. Validators
+stake their BNB tokens to participate in the network and earn rewards.
+
+However, validators also face the risk of losing their stakes if they behave in ways that could harm the network's
+integrity and reliability. This is where the slashing mechanism comes in. The slashing mechanism is a set of rules and
+functions implemented in the `StakeHub` contract 
+(which is a system contract and the address is `0x0000000000000000000000000000000000002002`) 
+that penalizes validators for violating certain conditions. The
+slashing mechanism is crucial for maintaining the BSC network's security, reliability, and integrity.
+
+The slashing mechanism covers three types of offenses: downtime, double signing, and malicious voting. Each offense has
+a different severity and penalty, depending on the impact it has on the network. In this document, we will explain the
+slashing conditions and mechanisms for each offense in detail.
+
+## Downtime Slash
+
+Validators are expected to maintain high availability to ensure the network's smooth operation. Validators failing to
+meet these uptime requirements are subject to slashing.
+
+- The `StakeHub` contract tracks validator uptime by measuring the number of blocks they sign within a certain window.
+- If a validator fails to sign at least a minimum percentage of blocks within the window, they are considered offline.
+- If a validator is offline for an extended period, beyond the acceptable thresholds defined in the contract, they will
+  have their stakes slashed by a predetermined amount.
+- The penalty for downtime is less severe than for double signing but is designed to encourage validators to maintain
+  the necessary infrastructure for high availability.
+
```

Comment:
Please paste the default slash params, like how many BNB slash and how long in jail

Reply:
added